web/public/locales/en.json

```diff
@@ -775,6 +778,24 @@
     "IM": {
       "zoneName": "Isle of Man"
     },
+    "IN": {
+      "zoneName": "India"
+    },
+    "IN-EA": {
+      "zoneName": "Eastern"
```

Comment:
Should we use the style we use for Italy here or the one we use for Norway, Sweden, and Denmark?
I.e. should it be:
```suggestion
      "zoneName": "Eastern India"
```

Reply:
We're a bit inconsistent here, as Italy for example don't have it:
<img width="266" alt="Screenshot 2023-01-10 at 22 17 50" src="https://user-images.githubusercontent.com/3296643/211664484-2a24ceed-47a4-40d9-804e-385c3bd22411.png">

I personally think it looks better like this (the "North Sweden (Sweden)" part is weird), but we then do need to make sure that the country is always included next to any usage - so I guess that option is at least safer and easier to use :)